docs/op-guide/deploy-tikv-using-docker-compose.md

```diff
@@ -20,63 +20,143 @@ Make sure you have installed the following items on your machine:
     sudo yum install docker docker-compose
     ```
 
-- Helm
-
-    ```bash
-    curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
-    ```
-
 - Git
 
     ```
     sudo yum install git
     ```
 
-## Install and deploy
+## Install 
 
-1. Download `tidb-docker-compose`.
+Download `tidb-docker-compose`.
 
-    ```bash
-    git clone https://github.com/pingcap/tidb-docker-compose.git
+```bash
+git clone https://github.com/pingcap/tidb-docker-compose.git
+```
+
+## Prepare cluster
+
+Here we will run a simple cluster with only 1 PD and 1 TiKV. The base docker compose config is:
+
+```yaml
+version: '2.1'
+
+services:
+  pd0:
+    image: pingcap/pd:latest
+    ports:
+      - "2379"
+    volumes:
+      - ./config/pd.toml:/pd.toml:ro
+      - ./data:/data
+      - ./logs:/logs
+    command:
+      - --name=pd0
+      - --client-urls=http://0.0.0.0:2379
+      - --peer-urls=http://0.0.0.0:2380
+      - --advertise-client-urls=http://pd0:2379
+      - --advertise-peer-urls=http://pd0:2380
+      - --initial-cluster=pd0=http://pd0:2380
+      - --data-dir=/data/pd0
+      - --config=/pd.toml
+      - --log-file=/logs/pd0.log
+    restart: on-failure
+  
+  tikv0:
+    image: pingcap/tikv:latest
+    volumes:
+      - ./config/tikv.toml:/tikv.toml:ro
+      - ./data:/data
+      - ./logs:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv0:20160
+      - --data-dir=/data/tikv0
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv0.log
+    depends_on:
+      - "pd0"
+    restart: on-failure
+```
+
+All the following example docker compose config file will contain this base config.
+
+## Example
+
+### [YCSB](https://github.com/pingcap/go-ycsb)
```

Comment:
-> Run [YCSB](https://github.com/pingcap/go-ycsb) to connect to the TiKV cluster:

Reply:
is it too long for a title?

Reply:
Nope. Complete titles are good for search engines.